pysmt/factory.py

```diff
@@ -441,6 +441,31 @@ def get_model(self, formula, solver_name=None, logic=None):
                 retval = solver.get_model()
             return retval
 
+    def get_implicant(self, formula, solver_name=None,
+                      logic=None):
+        mgr = self.environment.formula_manager
+        if logic == AUTO_LOGIC:
+            logic = get_logic(formula, self.environment)
+
+        with self.Solver(name=solver_name, logic=logic) \
+             as solver:
+            solver.add_assertion(formula)
+            check = solver.solve()
+            if not check:
+                return None
+            else:
```

Comment:
Which would be the semantics of a get_implicant at solver level? Note that the semantics "implicant of all the asserted formulae" would require us to keep the set of assertions in all the solvers...
Or was you suggesting a static method of Solver?